src/i18n/zh-cn/nav.ts

```diff
@@ -1,40 +1,57 @@
-import { NavDictionary } from '../translation-checkers';
+/**
+ * This configures the navigation sidebar.
+ * All other languages follow this ordering/structure and will fall back to
+ * English for any entries they haven’t translated.
+ *
+ * - All entries MUST include `text` and `key`
+ * - Heading entries MUST include `header: true` and `type`
+ * - Link entries MUST include `slug` (which excludes the language code)
+ */
+export default [
+	{ text: '由此开始', header: true, type: 'learn', key: 'startHere' },
+	{ text: '入门指南', slug: 'getting-started', key: 'getting-started' },
+	{ text: '安装', slug: 'install/auto', key: 'install' },
+	{ text: '编辑器配置', slug: 'editor-setup', key: 'editor-setup' },
+	{ text: '迁移指南', slug: 'migrate', key: 'migrate' },
 
-export default NavDictionary({
```

Comment:
We do want to keep using the `NavDictionary` function like this — it helps us check all the entries match up with the English navigation menu. Also means you don’t need all the `slug`, `key`,  etc. just the translation as this file had it before.